### DISCUSSION TITLE
scrolled frame wont work in notebook tab

### DISCUSSION THREAD

**Discussion by pissyonky** [israel-dryer/ttkbootstrap#225] [Q&A] [answered]
### Problem:
placing widgets in a scrolled notebook frame not working.

### This is the code 

```python
import ttkbootstrap as ttk
from ttkbootstrap.constants import *
from ttkbootstrap.scrolled import ScrolledFrame

app = ttk.Window()
app.geometry("400x300")

notebook = ttk.Notebook(app)

log_frame = ScrolledFrame(notebook, autohide=True)

notebook.add(log_frame.container, text="Log ")
notebook.place(x=40, y=30, height=200, width=300)

y = 10
for x in range(20):
	t = ttk.Checkbutton(log_frame.container, text=f"Checkbutton {x}")
	t.place(x=10, y=y)
	y += 30

app.mainloop() 
```

**Comment by israel-dryer** [comment, marked as answer]
Set the button master to the scrolled widget instead of the scrolled widget container.

The container is only used in the case that you are adding the scrolled widget to a notebook or a PanedWindow. See the example here: https://ttkbootstrap.readthedocs.io/en/latest/api/scrolled/scrolledframe/

**Reply by pissyonky** [reply]
When removing .container from the button master the button wont show.

### altered code

```python
import ttkbootstrap as ttk
from ttkbootstrap.constants import *
from ttkbootstrap.scrolled import ScrolledFrame

app = ttk.Window()
app.geometry("400x300")

notebook = ttk.Notebook(app)

log_frame = ScrolledFrame(notebook, autohide=True)

notebook.add(log_frame.container, text="Log ")
notebook.place(x=40, y=30, height=200, width=300)

y = 10
for x in range(20):
	t = ttk.Checkbutton(log_frame, text=f"Checkbutton {x}")
	t.place(x=10, y=y)
	y += 30

app.mainloop()
``` 

### Result
![result](https://user-images.githubusercontent.com/97931467/158041577-ca88b9d1-bfe5-4919-bb2c-9f67e2e52f55.PNG)

**Reply by israel-dryer** [reply]
Ok. The issue here is with place. In tkinter, unlike with grid and pack, place does not update the parent widget size to contain its children, so you have to give the frame an actual width and height for place to work in a frame.

However, I noticed that this was not working because I am not passing the width and height arguments to the internal frame. So, I need to push a fix out for that this week after running some tests. However, the solution will be to add `height=400, width=600` to the ScrolledFrame constructor (or however big you want the frame to be).

I'll let you now when I push an update.

**Reply by pissyonky** [reply]
Thanks for your post's I edited the scrolled.py file and everything is working perfect now. Thank you for your nice work.
